include/eve/detail/function/broadcast.hpp

```diff
@@ -40,13 +40,13 @@ namespace eve
   //!
   //! **Parameters**
   //!
-  //! `x`:  An instance of an \ref simd_value
+  //! `x`:  An instance of an [simdvalue](@ref simd_value)
   //!
   //! `i`:  An eve::index indicating which lane of `x` to broadcast
   //!
   //! **Return value**
   //!
-  //! If `x` is an instance of an [SIMD value](@ref simd_value) `T`, the call is equivalent to `T{x.get(I)}`.
+  //! If `x` is an instance of an [simd value](@ref simd_value) `T`, the call is equivalent to `T{x.get(I)}`.
```

Comment:
why the change ?

Reply:
it's the only place where it is in capital